packages/terra-modal/tests/nightwatch/modal-spec.js

```diff
@@ -13,136 +13,121 @@ module.exports = {
-      .waitForElementPresent('.terra-Modal', 1000)
-      .expect.element('.terra-Modal').to.have.attribute('class').which.contains('terra-Modal modal-custom-class');
+      .expect.element('div[role="document"]').to.have.attribute('class').which.contains('modal-custom-class');
   },
 
   'append css class to overlay': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/append-class`)
-      .waitForElementPresent('.terra-Modal', 1000)
-      .expect.element('.terra-Modal-overlay').to.have.attribute('class').which.contains('terra-Modal-overlay overlay-custom-class');
+      .expect.element('[class*="ModalOverlay"]').to.have.attribute('class').which.contains('overlay-custom-class');
   },
 
   'content overflow example': (browser) => {
     browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .waitForElementPresent('.terra-Modal', 1000);
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`);
   },
 
   'modal is focusable': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/no-focusable-content`)
-      .waitForElementPresent('.terra-Modal', 1000)
-      .expect.element('.terra-Modal').to.have.attribute('tabindex').which.equals('0');
+      .expect.element('div[role="document"]').to.have.attribute('tabindex').which.equals('0');
   },
 
   'background does not scroll when modal is open': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .waitForElementPresent('.terra-Modal', 1000)
       .expect.element('html').to.have.attribute('style').which.equals('overflow: hidden;');
 
     browser
-      .click('.terra-Modal-overlay')
-      .waitForElementNotPresent('.terra-Modal', 1000)
+      .click('[class*="ModalOverlay"]')
       .expect.element('html').to.have.attribute('style').which.equals('');
   },
 
   'isScrollable default value': (browser) => {
     browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .waitForElementPresent('.terra-Modal', 1000)
-      .expect.element('.terra-Modal').to.have.attribute('class').which.equals('terra-Modal');
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`);
   },
 
   'isScrollable={false}': (browser) => {
     browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/scrollable-false`)
-      .waitForElementPresent('.terra-Modal', 1000)
-      .expect.element('.terra-Modal').to.have.attribute('class').which.equals('terra-Modal');
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/scrollable-false`);
   },
 
   'isScrollable={true}': (browser) => {
```

Comment:
When we talked on Tuesday it sounded like we still wanted to `assert.elementPresent` on all tests to confirm tests are passing. Is that still the case?